packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# 48.21.0 - June 18, 2020
+
+## Added
+
+#### docs
+
+- Japanese translations for documentation and messages. ([PR #2246](https://github.com/forcedotcom/salesforcedx-vscode/pull/2246)) -Contribution by [shunkosa](https://github.com/shunkosa)
+
+- Added Mobile Preview Selection for `SFDX: Preview Component Locally`. See [Local Development (Beta)](https://developer.salesforce.com/tools/vscode/en/lwc/local-dev) for more details. ([PR #2266](https://github.com/forcedotcom/salesforcedx-vscode/pull/2266)) -Contribution by [brandonpage](https://github.com/brandonpage)
+
+#### salesforcedx-vscode-core
+
+- Added support for Aura and LWC bundle types for beta deploys ([PR #2270](https://github.com/forcedotcom/salesforcedx-vscode/pull/2270))
```

Comment:
```suggestion
- Support for Aura and LWC deploys in [Performance Enhancements](https://developer.salesforce.com/tools/vscode/en/user-guide/perf-enhancements) ([PR #2270](https://github.com/forcedotcom/salesforcedx-vscode/pull/2270))
```

or something like that ? I also noticed we didn't update that doc file so I'll open a PR for it in a min